src/base.ts

```diff
@@ -1,15 +1,21 @@
-import {Command} from '@heroku-cli/command'
+import {APIClient, Command} from '@heroku-cli/command'
 import {IConfig} from '@oclif/config'
+import * as url from 'url'
 
 export default abstract class extends Command {
+  shogun: APIClient
+
   protected constructor(argv: string[], config: IConfig) {
     super(argv, config)
 
-    this.heroku.defaults.host = process.env.HEROKU_DATA_HOST || 'postgres-api.heroku.com'
-    this.heroku.defaults.headers = {
+    const client = new APIClient(this.config, {})
+    const host = process.env.HEROKU_ADDONS_HOST
```

Comment:
We can't keep `HEROKU_DATA_HOST` ?

Reply:
ah shoot, I missed this. Going to fix it